<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CColorSelector</class>
 <widget class="QFrame" name="CColorSelector">
  <property name="maximumSize">
   <size>
    <width>140</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Color selector</string>
  </property>
  <property name="whatsThis">
   <string>select a color</string>
  </property>
  <layout class="QHBoxLayout" name="hl_ColorSelector">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="le_Color">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
     <property name="placeholderText">
      <string>hex or name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tb_ColorDialog">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_ColorIcon">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_Color</tabstop>
  <tabstop>tb_ColorDialog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
